DataFormats/CTPPSReco/interface/CTPPSDiamondLocalTrack.h

```diff
@@ -0,0 +1,94 @@
+/****************************************************************************
+ *
+ * This is a part of CTPPS offline software.
+ * Authors:
+ *   Laurent Forthomme (laurent.forthomme@cern.ch)
+ *   Nicola Minafra nicola.minafra@cern.ch)
+ *
+ ****************************************************************************/
+
+#ifndef DataFormats_CTPPSReco_CTPPSDiamondLocalTrack
+#define DataFormats_CTPPSReco_CTPPSDiamondLocalTrack
+
+#include "DataFormats/Math/interface/Point3D.h"
+#include "DataFormats/CTPPSReco/interface/CTPPSDiamondRecHit.h"
+
+//----------------------------------------------------------------------------------------------------
+
+class CTPPSDiamondLocalTrack
+{
+  public:
+    CTPPSDiamondLocalTrack() :
+      chi_squared_( 0. ), valid_( true ), t_( 0. ), t_sigma_( 0. ), mh_( 0 ) {}
+    virtual ~CTPPSDiamondLocalTrack() {}
+
+    //--- spatial get'ters
+
+    inline float getX0() const { return pos0_.x(); }
+    inline float getX0Sigma() const { return pos0_sigma_.x(); }
+
+    inline float getY0() const { return pos0_.y(); }
+    inline float getY0Sigma() const { return pos0_sigma_.y(); }
+
+    inline float getZ0() const { return pos0_.z(); }
+
+    inline float getChiSquared() const { return chi_squared_; }
+    
+    //--- spatial set'ters
+
+    inline void setX0( const float& x0 ) { pos0_.SetX( x0 ); }
```

Comment:
is there any practical reason to set X,Y,Z separately?
The interface can be simplified substantially by returning pos vector itself.